mne_bids/read.py

```diff
@@ -73,6 +73,30 @@ def _read_raw(raw_fpath, electrode=None, hsp=None, hpi=None, config=None,
     return raw
 
 
+def _handle_info_reading(sidecar_fname, raw):
+    """Read associated sidecar.json and populate raw.
+
+    Handle PowerLineFrequency of recording.
+    """
+    with open(sidecar_fname, "r") as f:
```

Comment:
I don't know if it applies here, but Mainak and Alex have repeatedly told me not to use single-letter variables, because they are hard to grep/search for ... I usually use `fin` or `fout` 
```suggestion
    with open(sidecar_fname, "r") as fin:
```

# of course, you'd have to update the line below as well.